integr8ly/tutorial-web-app
# APPDUX-81: Add role info under user menu

## Motivation
https://issues.redhat.com/browse/APPDUX-81

## What
Add login name, admin/dev icon, and admin/dev 'badge' based on whether the current user has admin rights or not, according to the design spec.

## Why
To better display whether the currently logged in user has admin account privileges or not.

## Verification Steps
No cluster to test on, so you will need to verify by running locally:
1. Start solution explorer and click the user menu in the masthead. It should display the administrator version of the menu.
2. Change ln 230 of masthead.js to 'isAdmin = false'.
3. Click the user menu in the masthead. It should now display the developer version of the menu.

## Checklist:

## Progress

## Additional Notes
**User menu when logged in as Admin:**
![admin_user_menu](https://user-images.githubusercontent.com/39063664/81115638-0a0bd980-8ef2-11ea-8264-93f5970c9614.png)

**User menu when logged in as Dev:**
![dev_user_menu](https://user-images.githubusercontent.com/39063664/81115660-12fcab00-8ef2-11ea-8b10-9f38ff173bd4.png)
